setup.py

```diff
@@ -21,6 +21,7 @@
     },
     install_requires=[
         'wagtail>=4.1',
+        'wagtail_modeladmin>=1.0',
     ],
```

Comment:
I think this should be:
```suggestion
    install_requires=[
        'wagtail>=4.1',
    ],
    extras_require={
        'testing': [
            'wagtail-modeladmin>=1.0',
        ],
    },
```
and then we should add to the docs about including `wagtail-modeladmin` as another dependency if required.

Reply:
In the tox.ini is references the `testing` extras so may be the `test-requirements.txt` doesn't need that.

The main point here is that when I force installed Wagtail==4.1 I saw a warning about `wagtail-modeladmin` needing `wagtail>=5.1` and I'm thinking we'd see that if the client site was running wagtail earlier than 5.1

[wagtailmenus] has docs that recommend adding it as you own dependency rather than forcing it from the package.

What do you think?

Reply:
Yup, this all makes sense:

Applied suggestion here: https://github.com/torchbox-forks/wagtail-robots/pull/5/commits/fe74326b481c9f74067a608b3b381b5e50dd0c32

I updated the docs to recommend installing wagtail_modeladmin when using Wagtail 5.1+: https://github.com/torchbox-forks/wagtail-robots/pull/5/commits/7c1ec6955203c13af39285c6a9e904f4d2c5c47d